database/factories/AudioVisualItemFactory.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Database\Factories;
+
+use Illuminate\Database\Eloquent\Factories\Factory;
+use Jitterbug\Models\AudioVisualItem;
+use Jitterbug\Models\Collection;
+use Jitterbug\Models\Format;
+use Jitterbug\Models\AudioItem;
+
+class AudioVisualItemFactory extends Factory
+{
+    /**
+     * The name of the factory's corresponding model.
+     *
+     * @var string
+     */
+    protected $model = AudioVisualItem::class;
+
+    /**
+     * Define the model's default state.
+     *
+     * @return array
+     */
+    public function definition()
+    {
+        return [
+          'call_number' => 'FS-'.$this->faker->randomNumber(4),
```

Comment:
Why is call_number in this former here but a word in AudioItem?

Reply:
I have absolutely no idea. I'll make them both this form.